gandalf/src/main/java/io/github/btkelly/gandalf/checker/HistoryChecker.java

```diff
@@ -29,13 +31,27 @@
      * @param alert item to compare against the history
      * @return {@code true} if provided alert matches the history
      */
-    boolean contains(Alert alert);
+    boolean contains(@NonNull final Alert alert);
+
+    /**
+     * Saves the provided {@link OptionalUpdate} to shared preferences.
```

Comment:
aye probably should, I'm not crazy about the name "contains" either (I named it), but whatever it works for now